src/internal/packager2/pull.go

```diff
@@ -31,6 +33,11 @@ import (
 
 // Pull fetches the Zarf package from the given sources.
 func Pull(ctx context.Context, src, dir, shasum string, filter filters.ComponentFilterStrategy, publicKeyPath string, skipSignatureValidation bool) error {
+	l := logger.From(ctx)
+	start := time.Now()
+	defer func() {
```

Comment:
Is the goal here to print this message even when we error?

Reply:
yeah, it saves from having to branch out the same debug end several times to track the duration. Just experimenting with it, not attached to this way of doing so.

Reply:
I'd prefer it at the end of the function. I don't think we'd need to branch out for this, and I think it makes sense to avoid the done pull message when an error is returned.

Reply:
Ya makes sense, resolved in latest rev.